debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -5,6 +5,25 @@
  */
 package io.debezium.connector.mysql;
 
+import com.github.shyiko.mysql.binlog.event.deserialization.AbstractRowsEventDataDeserializer;
```

Comment:
Nothing actually has changed in this file. Let's revert this one completely then.

Reply:
Ok. Some of it is because of intellij which moved around the imports. Would clean these things up.